op-node/rollup/derive/channel_out_test.go

```diff
@@ -29,6 +29,26 @@ func TestChannelOutAddBlock(t *testing.T) {
 	})
 }
 
+// TestOutputFrameSmallMaxSize tests that calling [OutputFrame] with a small
+// max size that is below the fixed frame size overhead of 23, will return
+// an error.
+func TestOutputFrameSmallMaxSize(t *testing.T) {
+	cout, err := NewChannelOut()
+	require.NoError(t, err)
+
+	// Set the channel out frame details
```

Comment:
Nit: test also just works without setting this. And 0 is the default value of int types.

Reply:
yessir, just boilerplate from when i set it up - removing ✅